### DISCUSSION TITLE
[Feature] Choose preferred extension when deduplicating photos

### DISCUSSION THREAD

**Discussion by PerrinWaldock** [immich-app/immich#10964] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Allow users to choose a preferred file extension when reviewing duplicates (e.g. prioritize keeping .heic over .jpg). Default behavior appears to be "keep largest file", which is often lossy when keeping a .jpg over a .heic

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#10665
